src/test/java/SetTest.java

```diff
@@ -0,0 +1,55 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@DisplayName("Set Collection에 대한 학습 테스트")
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("요구사항 1")
+    public void size(){
+        int size = numbers.size();
+        assertThat(size).isEqualTo(3);
+    }
+
+    @Test
+    public boolean contains(int value){
```

Comment:
객체지향 생활 제조 원칙
`규칙 6: 모든 엔티티를 작게 유지한다.`
이 규칙을 훌륭하게 적용했습니다. 메서도 분리 잘하셨네요👍

다만 아래처럼 수정하시는게 좋지 않을까 생각됩니다. :)
```suggestion
    private boolean contains(int value){
```

그리고 한가지 더 욕심이 있다면 메서드 위치를 제일 밑으로
내려 주는게 좋을거 같아요.
코드도 책이나 신문과 같아서 위로부터 쭉 읽히는게 가독성이 좋답니다.😄